### DISCUSSION TITLE
What is this part called? signature_help?

### DISCUSSION THREAD

**Discussion by winkee01** [hrsh7th/nvim-cmp#940] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/49930785/165449477-5ed6179d-8dff-4336-8a6e-fc387d51db9d.png)

After I confirm the cmp selection and try to continue typing in arguments, I can still see it (which is great!), but sometimes it doesn't show, why? how does the thing work?

![image](https://user-images.githubusercontent.com/49930785/165449712-60a77e1e-de0f-41e9-b531-04661308139f.png)

**Comment by winkee01** [comment, marked as answer]
ok, I figured it out.

**Reply by smjonas** [reply]
Please write down your answer here in case someone else has the same question.

**Reply by winkee01** [reply]
I know that it comes from a plugin called `ray-x/lsp_signature.nvim`, but I don't know how it was triggered.
